<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>freeSearch</class>
 <widget class="QDialog" name="freeSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>661</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>Welcome, </string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Addresss: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="latlongInput">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>581</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(Format: 0.0, 0.0)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
